percona-xtradb-cluster-8.0-backup/run_backup.sh

```diff
@@ -166,4 +178,6 @@ fi
 if (($SST_FAILED == 0)); then
 	touch /tmp/backup-is-completed
 fi
+
+echo "Finished at $(date '+%Y-%m-%d %H:%M:%S')"
```

Comment:
Shouldn't we use `log INFO ....`?

Reply:
changed to `log INFO` and removed `date` as `log` automatically adds date and time